addons/html_builder/static/src/core/plugins/core_builder_action_plugin.js

```diff
@@ -48,127 +111,95 @@ const styleMap = {
-        isApplied: ({ editingElement, param: attributeName, value }) => {
-            if (value) {
-                return editingElement.dataset[attributeName] === value;
-            } else {
-                return !(attributeName in editingElement.dataset);
-            }
-        },
-        apply: ({ editingElement, param: attributeName, value }) => {
-            if (value) {
-                editingElement.dataset[attributeName] = value;
-            } else {
-                delete editingElement.dataset[attributeName];
-            }
-        },
-        clean: ({ editingElement, param: attributeName }) => {
+    clean: ({ editingElement, param: attributeName }) => {
+        editingElement.removeAttribute(attributeName);
+    },
+};
+
+const dataAttributeAction = {
+    getValue: ({ editingElement, param: attributeName }) => editingElement.dataset[attributeName],
+    isApplied: ({ editingElement, param: attributeName, value }) => {
+        if (value) {
+            return editingElement.dataset[attributeName] === value;
+        } else {
+            return !(attributeName in editingElement.dataset);
+        }
+    },
+    apply: ({ editingElement, param: attributeName, value }) => {
+        if (value) {
+            editingElement.dataset[attributeName] = value;
+        } else {
             delete editingElement.dataset[attributeName];
-        },
+        }
     },
+    clean: ({ editingElement, param: attributeName }) => {
+        delete editingElement.dataset[attributeName];
+    },
+};
 
-    // TODO maybe find a better place for this
-    setClassRange: {
-        getValue: ({ editingElement, param: classNames }) => {
-            for (const index in classNames) {
-                const className = classNames[index];
-                if (editingElement.classList.contains(className)) {
-                    return index;
-                }
+// TODO maybe find a better place for this
+const setClassRange = {
+    getValue: ({ editingElement, param: classNames }) => {
+        for (const index in classNames) {
+            const className = classNames[index];
+            if (editingElement.classList.contains(className)) {
+                return index;
             }
-        },
-        apply: ({ editingElement, param: classNames, value: index }) => {
-            for (const className of classNames) {
-                if (editingElement.classList.contains(className)) {
-                    editingElement.classList.remove(className);
-                }
+        }
+    },
+    apply: ({ editingElement, param: classNames, value: index }) => {
+        for (const className of classNames) {
+            if (editingElement.classList.contains(className)) {
+                editingElement.classList.remove(className);
             }
-            editingElement.classList.add(classNames[index]);
-        },
+        }
+        editingElement.classList.add(classNames[index]);
     },
 };
```

Comment:
Check gradient: + dots in timeline + gradient and come back in Custom

Reply:
It s not supported in website